index.html

```diff
@@ -5,9 +5,158 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+ <link rel="stylesheet" href="/style.css">
+ <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=lato">
+ <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+ <script src="https://kit.fontawesome.com/7967e81f77.js" crossorigin="anonymous"></script>
   <title>Document</title>
 </head>
 <body>
+  <main>
+    <section class="home">
+  <header class="nav-header">
+    <nav class="navItems">
+      <a class="logo navbar-brand" href="/index.html"> <img src="/icons/kamzzylearns.png" alt="logo"> </a>
+    <ul class="nav-menu">
+     <li><a class="port-nav nav-link" href="/about.html">About</a></li>
+     <li><a class="about-nav nav-link" href="#">Join</a></li>
+     <li><a class="contact-nav nav-link" href="#">Contact</a></li>
+     </ul>
+     <div class="bars">
+      <span class="bar-span"></span>
+      <span class="bar-span"></span>
+      <span class="bar-span"></span>
+  </div>
+ </nav>
+</header>
+
+<h3 class="home-h3">"Hello! Tech Enthusiasts"</h3>
+<div class="home-div">
+  <h1>AUGUMENTED REALITY</h1>
+  <h1>SPARK TRAINING</h1>
+  <h1 class="date">2022</h1>
+</div>
+<div class="content">
+  <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Atque quisquam cum fugit nulla rerum aliquam quia commodi debitis obcaecati sequi ut vero, aut rem sint totam dolorum laborum accusantium suscipit! </p>
+</div>
+<h1>2022.05.27(FRI) ~ 28(SAT)</h1>
+<p>@ Kamzzy Creative Tech Hub Lagos State Nigeria</p>
+</section>
+  <section class="program">
+    <div class="checkDiv">
+    <input type="checkbox" class="toggle" id="toggle">
+    <label for="toggle">Main Program</label> 
```

Comment:
- [ ] It would be cool if you use an `hx` tag for this title instead of a `label` tag, so this way we can be more semantic.
- [ ] Also, it would be nice if you can center your text, so we can be a bit more organized here.

Reply:
@Lordkaito you said I should use a hx tag instead of a label for my checkbox, kindly suggest how to implement that.